pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
> Actually, with the multi module effort and what I started about building assembly modules, this won't work as conf, lib ... won't be part anymore of core project.

Can you elaborate on this? This is exactly why i was running mvn exec:exec in the first place. If these folders are unnecessary, perhaps we should remove them in an independent PR before we push refactoring? I am concerned about them.